pkg/gatherers/clusterconfig/clusterconfig_gatherer.go

```diff
@@ -67,6 +67,7 @@ var gatheringFunctions = map[string]gatheringFunction{
 	"crds":                              importantFunc((*Gatherer).GatherCRD),
 	"host_subnets":                      importantFunc((*Gatherer).GatherHostSubnet),
 	"machine_sets":                      importantFunc((*Gatherer).GatherMachineSet),
+	"machine_configs":                   failableFunc((*Gatherer).GatherMachineConfigs),
```

Comment:
Should it be failable if we're gonna collect all of them?

Reply:
Well I think most of our gatherers should be "failable", but then there's of course the question whether we need this flag or not. This flag is only saying that if the specific gatherer failed and is marked as important then repeat the gathering cycle in some shorter time. Given this, I don't consider `MachineConfig` resource so important.

Reply:
Makes sense in that context.